salt/cdh/templates/cfg_production.py.tpl

```diff
@@ -158,7 +164,12 @@ MAPRED_CFG = {
                     'resourcemanager_config_safety_valve':
                         '<property> \r\n<name>yarn.resourcemanager.proxy-user-privileges.enabled</name>\r\n<value>true</value>\r\n</property>',
                     'resource_manager_java_heapsize': '4294967296',
-                    'resource_manager_log_dir': '/var/log/pnda/hadoop-yarn'
+                    'resource_manager_log_dir': '/var/log/pnda/hadoop-yarn',
+                    'yarn_scheduler_increment_allocation_mb': '1024',
+                    'yarn_scheduler_maximum_allocation_mb': '65536',
```

Comment:
Why not have yarn_scheduler_maximum_allocation_mb equal to yarn_nodemanager_resource_memory_mb ?

Reply:
yarn_scheduler_maximum_allocation_mb is the maximum one container can be, there's 2 CPUs of 12 cores and 128GB of memory, therefore the directly addressable memory from one process is 64GB, being the highest multiple less than 77GB
also, does it make sense to have one Java process greater than 64GB heap? probably not

Reply:
64GB seems like a reasonable maximum for one container at this scale - will review following testing